src/test/resources/test.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE `bertalan-bodroghelyi-sandbox.tf_test.tf_test3`
```

Comment:
use a placeholder here for the project, which should be replaced runtime from the context